packages/ag-charts-community/src/chart/series/series.ts

```diff
@@ -351,12 +354,12 @@ export abstract class Series<
         this._pickNodeCache.clear();
     }
 
-    setOptionsData(input: unknown[]) {
+    setOptionsData(input: DataSet | undefined) {
         this._data = input;
         this.onDataChange();
     }
 
-    setChartData(input: unknown[]) {
+    setChartData(input: DataSet | undefined) {
```

Comment:
Could it take the array as before and call `DataSet.wrap()` itself, to save repeating it elsewhere?

Reply:
No, we need the `DataSet` reference to be identical for deduplication purposes in `DataController`.